python/cudf_kafka/setup.py

```diff
@@ -0,0 +1,109 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+import os
+import shutil
+import sysconfig
+from distutils.sysconfig import get_python_lib
+
+import numpy as np
+from Cython.Build import cythonize
+from setuptools import find_packages, setup
+from setuptools.extension import Extension
+
+import versioneer
+
+install_requires = ["cudf", "cython"]
+
+cython_files = ["cudf_kafka/_lib/*.pyx"]
+
+CUDA_HOME = os.environ.get("CUDA_HOME", False)
+if not CUDA_HOME:
+    path_to_cuda_gdb = shutil.which("cuda-gdb")
+    if path_to_cuda_gdb is None:
+        raise OSError(
+            "Could not locate CUDA. "
+            "Please set the environment variable "
+            "CUDA_HOME to the path to the CUDA installation "
+            "and try again."
+        )
+    CUDA_HOME = os.path.dirname(os.path.dirname(path_to_cuda_gdb))
+
+if not os.path.isdir(CUDA_HOME):
+    raise OSError(f"Invalid CUDA_HOME: directory does not exist: {CUDA_HOME}")
+
+cuda_include_dir = os.path.join(CUDA_HOME, "include")
+
+CUDF_ROOT = os.environ.get("CUDF_ROOT", "../../cpp/build/")
```

Comment:
Similarly will likely want a `CUDF_KAFKA_ROOT` defined here for use in rapids-compose, cc @trxcllnt @millerhooks

Reply:
Makes sense. I will add that